README.md

```diff
@@ -226,6 +226,41 @@ Follow Helm Chart installation instructions [here](https://docs.openshift.com/co
 
 ## Additional information
 
+### Additional Workflow Namespaces
+
+If you want to deploy workflows on other namespaces, make sure you follow these steps:
+
+ - For the cluster Sonataflow services to accept the traffic from the workflows, set this label:
+   ```console
+      oc label ns my-additional-workflow-ns rhdh.redhat.com/workflow-namespace=""
+   ```
+
+ - Set a network policy to allow only RHDH and Sonatflow services traffic. Taken from the `charts/orchestrator/templates/network-policy.yaml`
```

Comment:
does it means that this guide can be outdated? 
Maybe we can add a link instead?

Reply:
prblem is that the branch would be hardcoded here... I am not very happy with that. Because this specifc gh-pages branch corresponds to upstream/main, and when we move to a stable branch then this will have to be changed to the stable branch. I'm unsure what is best here. Any more ideas?